inz-demo/revocable-id/Issues/id/description.xml

```diff
@@ -0,0 +1,41 @@
+<IssueSpecification version="4">
+	<Name>
+		<en>Demo Revocable ID</en>
+		<nl>Demo Revocable ID</nl>
+	</Name>
+	<ShortName>
+		<en>Demo Revocable ID</en>
+		<nl>Demo Revocable ID</nl>
+	</ShortName>
+	<SchemeManager>inz-demo</SchemeManager>
+	<IssuerID>revocable-id</IssuerID>
+	<CredentialID>id</CredentialID>
+	<Description>
+		<en>Demo Revocable ID</en>
+		<nl></nl>
+	</Description>
+	<ShouldBeSingleton>true</ShouldBeSingleton>
+	<DisallowDelete>true</DisallowDelete>
```

Comment:
Disallowing delete for any special reason?

Reply:
Not really! I'm gonna remove it.